xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuMacro.xml

```diff
@@ -658,22 +713,35 @@ $menuTemplateDoc.content.replace('{', '')
     .box-shadow(0 2px 8px rgba(0,0,0,0.4) inset);
     min-height: @navbar-height;
     padding-left: 25px;
+    .xDropdown.open {
+      &gt; .xDropdown-header &gt; .xDropdown-header-toggle:before {
+        transform: rotate(0);
+      }
+      &gt; ul {
+        display: block;
+      }
+    }
     &amp; &gt; ul {
       padding-left: 0;
       list-style-type: none;
       margin: 0;
+      min-height: 50px;
+      display: flex;
+      align-items: stretch;
       &amp; &gt; li {
         position: relative;
-        display: block;
+        min-height: 50px;
+        display: flex;
+        align-items: center;
         padding: @nav-link-padding;
-        padding-top: @navbar-padding-vertical;
-        padding-bottom: @navbar-padding-vertical;
+        padding-top: 0px;
+        padding-bottom: 0px;
```

Comment:
iirc what a size is zero it's better to skip the unit.
```suggestion
        padding-top: 0;
        padding-bottom: 0;
```

Reply:
I'm curious about the rationale @manuelleduc why is it better?

Reply:
Committing changes with similar changes at L667 & L676 :+1:

Reply:
> I'm curious about the rationale @manuelleduc why is it better?

The unit is optional when the value is 0. But :
1. I don't think we have a code style for that (yet).
2. afaics there is no de-facto norm in our code base

PS: I think I took this habit because IntelliJ is warning about this when editing CSS.

Reply:
ok thanks. I've read quickly https://stackoverflow.com/questions/7923042/when-specifying-a-0-value-in-css-should-i-explicitly-mark-the-units-or-omit and most people seem to agree with no unit.